### DISCUSSION TITLE
The back-end is throwing 500s when I try to add environment variables to my project

### DISCUSSION THREAD

**Discussion by jeffthompsonCC** [vercel/community#7028] [Bug Report]
### Page to Investigate

https://vercel.com/change-captain/cc-client/settings/environment-variables

### Steps to Reproduce

Looks like a DEV left the React dev tools for Context in the code and it's that that is throwing the error:
The back-end throws a 500 when i try to add a new variable
endpoint: 
Request URL:
https://vercel.com/api/v10/projects/prj_c0GeB2TXV0JYohdTG5bcOQ7srQFA/env?teamId=team_U9SkGd6EiuFEIruNFYPuKLYD
Request Method:
POST
Status Code:
500 Internal Server Error
Remote Address:
76.76.21.9:443

I see this in the console for front-end logging:
TypeError: Cannot read properties of undefined (reading '__reactContextDevtoolDebugId')
    at <anonymous>:10:1513
    at c (<anonymous>:10:1837)
    at e (<anonymous>:10:1913)
    at e (<anonymous>:10:1931)
    at e (<anonymous>:10:1931)
    at i (<anonymous>:10:1942)
    at t.onCommitFiberRoot.t.onCommitFiberRoot (<anonymous>:10:3098)
    at 78dffda6-fc7ad7a6711b3ddc.js?dpl=dpl_C27n1nwaWUYqK3Vr3ge1dxyCDqZL:3:121124
    at 78dffda6-fc7ad7a6711b3ddc.js?dpl=dpl_C27n1nwaWUYqK3Vr3ge1dxyCDqZL:3:121193
    at uu (78dffda6-fc7ad7a6711b3ddc.js?dpl=dpl_C27n1nwaWUYqK3Vr3ge1dxyCDqZL:3:121414)
    at i2 (78dffda6-fc7ad7a6711b3ddc.js?dpl=dpl_C27n1nwaWUYqK3Vr3ge1dxyCDqZL:3:106883)
    at iJ (78dffda6-fc7ad7a6711b3ddc.js?dpl=dpl_C27n1nwaWUYqK3Vr3ge1dxyCDqZL:3:106027)

**Comment by thiagomorales** [comment]
Same here. Either add/get envs using the Vercel console or trying to pull the envs using the Vercel CLI.

<img width="521" alt="image" src="https://github.com/vercel/community/assets/4661217/52ff1711-1353-40ee-9f86-faad45600176">

**Reply by thiagomorales** [reply]
> Same here. Either add/get envs using the Vercel console or trying to pull the envs using the Vercel CLI.
> 
> <img alt="image" width="521" src="https://private-user-images.githubusercontent.com/4661217/334909704-52ff1711-1353-40ee-9f86-faad45600176.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcwMDczNTksIm5iZiI6MTcxNzAwNzA1OSwicGF0aCI6Ii80NjYxMjE3LzMzNDkwOTcwNC01MmZmMTcxMS0xMzUzLTQwZWUtOWY4Ni1mYWFkNDU2MDAxNzYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjlUMTgyNDE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTViNTY5NGI1NzIyMzU3NmViYzZjOTQ2MmJiYjgzNWFiNjM0NzI1ZTU0NmQxNzc3ODdjOGM4NWVhMDE0YTcxZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Kw5M4XOk1glw_QFt4ilzIvb93G9ToszUdmRr1qDnmpA">

An incident related to this has been opened.
https://www.vercel-status.com/incidents/ddgdx4h1w58p
